hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/detailslayout/AbstractTableDetailsLayout.java

```diff
@@ -53,7 +57,7 @@
     @Autowired
     protected SpPermissionChecker permissionChecker;
 
-    protected UI ui;
+    protected T selectedBaseEntity;
```

Comment:
Change it to private scope and introduce a protected getter-method. 
This is more readable than using the field directly in sub-classes